src/app/settings/shelf-settings/components/edit-shelf-action-panel-configuration/edit-shelf-action-panel-configuration.component.ts

```diff
@@ -51,4 +55,37 @@ export class EditShelfActionPanelConfigurationComponent implements OnInit {
   public cancel(): void {
     this.onCancel.emit()
   }
+
+  public get saveButtonErrorsTooltip(): string | undefined {
+    const invalidControlsLabels: string[] = this.findInvalidControlsLabels()
+    return invalidControlsLabels.length > 0 ? $localize`common.form-required-field.button-tooltip` + `: ${invalidControlsLabels.join(', ')}` : undefined
+  }
+
+  private findInvalidControlsLabels(): string[] {
+    const invalidControlLabels: string[] = []
+    const controls = this.form.controls
+    for (const name in this.form.controls) {
+      if (controls[name].valid) {
+        continue
+      }
+      const translateControl: string | undefined = this.translateControl(name)
+      if (translateControl) {
+        invalidControlLabels.push(translateControl)
+      }
+    }
+    return invalidControlLabels
+  }
+
+  private translateControl(controlName: string): string | undefined {
+    switch (controlName) {
+      case FORM_NAME_CONTROL_LABEL:
```

Comment:
From the contents of the `X_CONTROL_LABEL` variables it does not seem to be labels, but rather attribute names or identifiers. Perhaps they should be renamed to reflect that, e.g. `PANEL_NAME_CONTROL_ID` (The `FORM_` prefix seems redundant).

Reply:
Fixed